behat/features/register.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Register
+  In order to be able to register
+  As an anonymous user
+  We need to be able to have access to the register page
+
+  @api
+  Scenario: Anon auth user add smart agent
```

Comment:
IMO these tests look very scripty. They should deal with [behavior](http://www.gizra.com/content/behat-the-right-way/).

```
Given I am an anonymous user
When I register to the site
And Click on "מצאו לי עבודה!"
Then I should see "איך בא לכם לשלוח קורות חיים?"
```